src/main/resources/db/migration/V16_1__add_column_friskmeldingtilarbeidsformidling_fradato.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE person_oversikt_status
+    ADD COLUMN friskmelding_til_arbeidsformidling_fom DATE;
+
+CREATE INDEX IX_PERSON_OVERSIKT_FTA_FOM
```

Comment:
Litt usikker på hva slags eksekveringsplan postgres legger opp til for den utvidede spørringen og hvor nyttig denne indeksen vil være. Må følge med på databasesiden når det prodsettes sånn at vi sikrer at query'ene går omtrent like raskt som før.

Reply:
Må vi ikke oppdatere IX_PERSON_OVERSIKT_STATUS_ENHETENS_OVERSIKT også? Som vi pleier å gjøre når vi legger til et nytt felt det spørres på for å hente enhetens oversikt :cold_sweat:

Reply:
Det er ikke opplagt for meg at vi kan legge noe til i den eksisterende indeksen som vil løse dette siden det ikke er snakk om et BOOLEAN felt.

Reply:
Hm nei, det er sant. Vi får prøve dette da - og følge godt med på databasespørringene i prod. Kanskje gjøre endringen utenfor den mest travle arbeidstiden?